### PR TITLE
Notification Popups

### LINKED ISSUE #32: Notifications
- https://www.npmjs.com/package/react-toastify
Add notifications for success and errors

### PR DESCRIPTION
**Pull request by aitikgupta**
Resolves #32 

Also removed `formErrors`.

### CODE REVIEW

File: frontend/src/Components/Account/Register.js [hunk 1]
```diff
@@ -97,15 +97,16 @@ export default function Register() {
...
-          console.log(resp.data.message); //later we can change log to notifications
+          toast.success(resp.data.message);
           navigate("/login");
           setLoading(false);
         })
-        .catch((err) => {
-          setFormError(true);
+        .catch(({ response }) => {
+          if (response) {
+            toast.error("Status " + response.status + " (" + response.statusText + "): " + JSON.stringify(response.data.detail));
```
Comment (bodhisha) [inline frontend/src/Components/Account/Register.js]: From a user perspective, the user need not have to see the status code and other server stuff right? He just needs to know why the error occurred

I think  error detail alone is enough

File: frontend/src/Components/Account/Login.js [hunk 2]
```diff
@@ -24,13 +24,16 @@ export default function Login() {
...
         localStorage.setItem("access_token", resp.data.data.access_token);
         navigate("/home");
         setLoading(false);
         window.location.reload();
       })
-      .catch((err) => {
-        setFormError(true);
+      .catch(({ response }) => {
+        if (response) {
+          toast.error("Status " + response.status + " (" + response.statusText + "): " + JSON.stringify(response.data.detail));
```
Comment (bodhisha) [inline frontend/src/Components/Account/Login.js]: Same as in register.js

**Review by bodhisha — CHANGES_REQUESTED** [review]

File: frontend/src/Components/Account/Login.js (hunk 2 above)
Comment (aitikgupta) [reply frontend/src/Components/Account/Login.js]: Thing is, when you give a blank input to the backend (when the return detail is "Unprocessable Identity", it's very hard to understand it without knowing "Unprocessable Identity")
I can remove the response status, but I guess we can keep the `statusText`?

Comment (bodhisha) [reply frontend/src/Components/Account/Login.js]: `response.data.detail` specifies clearly the error na? 🤔

Comment (aitikgupta) [reply frontend/src/Components/Account/Login.js]: > `response.data.detail` specifies clearly the error na?

image

Comment (aitikgupta) [reply frontend/src/Components/Account/Login.js]: Done 👍🏼 
(Approve before merging. XD)

**Review by bodhisha — APPROVED** [review]
Cool.. 🎉

**Review by DebugAgrawal — APPROVED** [review]

**Pull request merged**